[2024-06-28T12:45:19.441+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: quarta_dag.tsk1 manual__2024-06-28T12:45:06.911173+00:00 [queued]>
[2024-06-28T12:45:19.449+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: quarta_dag.tsk1 manual__2024-06-28T12:45:06.911173+00:00 [queued]>
[2024-06-28T12:45:19.449+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-06-28T12:45:19.449+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-06-28T12:45:19.449+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-06-28T12:45:19.459+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): tsk1> on 2024-06-28 12:45:06.911173+00:00
[2024-06-28T12:45:19.465+0000] {standard_task_runner.py:55} INFO - Started process 224 to run task
[2024-06-28T12:45:19.467+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'quarta_dag', 'tsk1', 'manual__2024-06-28T12:45:06.911173+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/quartadag.py', '--cfg-path', '/tmp/tmpoe_7v93f']
[2024-06-28T12:45:19.468+0000] {standard_task_runner.py:83} INFO - Job 94: Subtask tsk1
[2024-06-28T12:45:19.509+0000] {task_command.py:388} INFO - Running <TaskInstance: quarta_dag.tsk1 manual__2024-06-28T12:45:06.911173+00:00 [running]> on host 90955aa900a6
[2024-06-28T12:45:19.561+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=quarta_dag
AIRFLOW_CTX_TASK_ID=tsk1
AIRFLOW_CTX_EXECUTION_DATE=2024-06-28T12:45:06.911173+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-28T12:45:06.911173+00:00
[2024-06-28T12:45:19.561+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-06-28T12:45:19.562+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 5']
[2024-06-28T12:45:19.572+0000] {subprocess.py:86} INFO - Output:
[2024-06-28T12:45:24.575+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-28T12:45:24.649+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=quarta_dag, task_id=tsk1, execution_date=20240628T124506, start_date=20240628T124519, end_date=20240628T124524
[2024-06-28T12:45:24.707+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-06-28T12:45:24.722+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
